docker-compose.yml

```diff
@@ -1,9 +1,8 @@
 version: "3.5"
+name: lakefs-with-jupyter
```

Comment:
IMHO the root Docker Compose should have the vanilla container names, _or_ we prefix all the containers in it with `lakeFS-samples` or s'thing like that.

Reply:
good idea, changed